src/isIsogram.test.js

```diff
@@ -8,12 +8,34 @@ describe(`Function 'isIsogram':`, () => {
   });
 
   it(`should return a boolean`, () => {
+    expect(typeof isIsogram('look')).toEqual('boolean');
+  });
 
+  it(`should be not case sensitive`, () => {
+    expect(isIsogram('cAt')).toBe(true);
   });
```

Comment:
Here you don't check if it's case sensitive because there are no same letters in different cases (like 'A'/'a').